DevEdu.API/DevEdu.DAL/Repositories/TaskRepository.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Data.SqlClient;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Dapper;
+using DevEdu.DAL.Models;
+
+namespace DevEdu.DAL.Repositories
+{
+    public class TaskRepository : BaseRepository, ITaskRepository
+    {
+        public TaskRepository()
+        {
+
+        }
+        public TaskDto GetTaskById(int id)
+        {
+            TaskDto task = _connection.QuerySingle<TaskDto>(
+                "dbo.Task_SelectById",

```

Comment:
each name of stored procedures must be placed somewhere as a constant
otherwise, it's an example of the "magic numbers" antipattern